regolith/helpers/a_proposalhelper.py

```diff
@@ -28,70 +23,57 @@ def subparser(subpi):
+    subpi.add_argument("-b", "--begin_date", nargs="+",
+                       help="The start date for the proposed grant in format YYYY-MM-DD."
                        )
-    subpi.add_argument("--submitted_date",
-                       help="Day on which the proposal is submitted in format YYYY-MM-DD"
+    subpi.add_argument("--cpp_flag", nargs="+",
+                       help="Current and pending form (true or false)"
                        )
-    subpi.add_argument("--due_date", nargs="+",
-                       help="The due date for the proposal in format YYYY-MM-DD."
+    subpi.add_argument("--cpp_agencies", nargs="+",
+                       help="Other agencies to which the proposal has been submitted"
                        )
-    subpi.add_argument("-d", "--duration",
-                       help="Duration of proposal in years"
+    subpi.add_argument("--cpp_institution", nargs="+",
+                       help="The institution to be stated on the current"
+                            "and pending form"
                        )
-    subpi.add_argument("--funder", nargs="+",
-                       help="Who funds the proposal. As funder in grants"
+    subpi.add_argument("--cpp_months_academic", nargs="+",
+                       help="Number of working months in the academic year"
+                            "to be stated on the current and pending form"
                        )
-    subpi.add_argument("--full", nargs="+",
-                       help="Full body of the proposal"
+    subpi.add_argument("--cpp_months_summer", nargs="+",
+                       help="Number of working months in the summer to be"
+                            "stated on the current and pending form"
                        )
-    subpi.add_argument("-n", "--notes", nargs="+",
-                       help="Anything to note"
+    subpi.add_argument("--cpp_scope", nargs="+",
+                       help="Scope of the project"
                        )
-    subpi.add_argument("--pi", "--principal_investigator", nargs="+",
-                       help="Name of principal investigator"
-                       )
-    subpi.add_argument("--pre", nargs="+",
-                       help="Information about pre-posal"
-                       )
-    subpi.add_argument("-s", "--status",
-                        help=f"status, from {ALLOWED_STATI}. default is accepted")
-    subpi.add_argument("-m", "--team_members", nargs="+",
-                       help="Information about the team members participating"
-                            "in the grant"
+    subpi.add_argument("-e","--end_date", nargs="+",
+                       help="The end date for the proposed grant in format YYYY-MM-DD."
                        )
-    subpi.add_argument("-t", "--title",
-                        help="Actual title of the proposal"
+    subpi.add_argument("-f", "--funder", nargs="+",
+                       help="Who funds the proposal. As funder in grants"
```

Comment:
"agency where the proposal is being submitted"

Reply:
done